src/languages/php.js

```diff
@@ -11,23 +11,105 @@ Category: common
  * @returns {LanguageDetail}
  * */
 export default function(hljs) {
+  const regex = hljs.regex;
+  const LABEL = '[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*' +
```

Comment:
Is this is a valid ident regex then we'd commonly name it `IDENT_RE`

Reply:
Changed